docs/howto/openstack/neutron/new-network-control-panel.md

```diff
@@ -1,57 +1,332 @@
-# Creating a new network from the Cleura Cloud Control Panel
+# Creating new networks on the Cleura Cloud
 
-Before creating a server in {{extra.brand}} Cloud, you need to have a
-network to make the new server a member of. You may have more than one
-network per region, so let us now walk through creating a new network
-using the {{extra.gui}}. Fire up your favorite web browser, navigate
-to the [Cleura Cloud](https://{{extra.gui_domain}}) page, and login
-into your {{extra.brand}} account.
+Before creating a server in {{extra.brand}} Cloud, you need at least
+one network to make the new server a member of. Since you may have
+more than one network per region, let us now walk through creating two
+new networks; one using the {{extra.gui}}, and another using the
+OpenStack CLI.
 
-On the top right-hand side of the {{extra.gui}}, click the _Create_
-button. A new pane will slide into view from the right-hand side of
-the browser window, titled _Create_.
+## Prerequisites
 
-![Create a new object](assets/new-net-panel/shot-01.png)
+Whether you choose to work from the {{extra.gui}} or with the
+OpenStack CLI, you need to [have an
+account](/howto/getting-started/create-account) in {{extra.brand}}
+Cloud. Additionally, to use the OpenStack CLI make sure to [enable it
+first](/howto/getting-started/enable-openstack-cli).
 
-You will notice several rounded boxes prominently displayed on that 
-pane, each for defining, configuring, and instantiating a different 
-{{extra.brand}} Cloud object. Go ahead and click the _Network_ box. A 
-new pane titled _Create Network_ will slide over. At the top, type in a 
-name and select one of the available regions for the new network.
+## Creating a network
 
-![New network name and region](assets/new-net-panel/shot-02.png)
+To create a network from the {{extra.gui}}, fire up your favorite web
+browser, navigate to the [Cleura Cloud](https://{{extra.gui_domain}})
+page, and login into your {{extra.brand}} account. On the other hand,
+if you prefer to work with OpenStack CLI, please do not forget to
+source the relative RC file first.
```

Comment:
Don't put this outside the content tab. One of these clearly applies to the GUI, the other to the CLI. They should be in their respective content tabs.

Reply:
I thought to use this paragraph as a "starter" for the whole section. Sort of saying something generic enough and useful, then spring into action.